game/resource/English/ability/items/tooltip_infinity_bottle.txt

```diff
@@ -3,5 +3,6 @@
 //=============================================================================
 "DOTA_Tooltip_Ability_item_infinite_bottle"                                         "Infinity Bottle"
 "DOTA_Tooltip_Ability_item_infinite_bottle_Description"                             "<h1>Active: Regenerate</h1>Consumes a charge to restore %health_restore% health and %mana_restore% mana over %restore_time% seconds.<br><br>Initial Charges: 3<br>Max Charges: Infinite"
+"DOTA_Tooltip_ability_item_bottle_Lore"                                             "in an attempt to destroy our world the Whitelord gave nutrals all bottles. As the flood of enchantments began crashing games the Darklord came and told the heroes "bottles now stack!" thus the infintiy bottle was born"
```

Comment:
Functional issues: You used "" as part of the tooltip, so your lore ends after heroes. unexpected token of bottles now stack and then a key of thus the infinity bottle was born.
Also it's the lore for vanilla bottle which you can no longer buy or drop.